.travis.yml

```diff
@@ -23,6 +23,7 @@ before_install:
  # Use sed to replace the SSH URL with the public URL, then initialize submodules
  - sed -i 's/git@github.com:/https:\/\/github.com\//' .gitmodules
  - git submodule update --init
+ - git clone https://github.com/osa1/concurrent-skiplist -b refactoring ./haskell/concurrent-skiplist
```

Comment:
Ok, I added you as a contributor to my copy of the repo -- so merge PRs or push at will.

Also, let's switch that to a proper submodule.  Or, if we ever put a direct `git clone` hack in a script, it should be fixed to a specific SHA.  (Which effectively simulates a submodule.)  Grabbing the tip of a branch is just.... icky.